Recrutify/Recrutify.Services/Validators/BulkCreateTestFeedbackValidator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentValidation;
+using Recrutify.DataAccess.Models;
+using Recrutify.DataAccess.Repositories.Abstract;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Validators
+{
+    public class BulkCreateTestFeedbackValidator : AbstractValidator<BulkCreateTestFeedbackDTO>
+    {
+        private readonly ICandidateRepository _candidateRepository;
+
+        public BulkCreateTestFeedbackValidator(ICandidateRepository candidateRepository)
+        {
+            _candidateRepository = candidateRepository;
+
+            RuleFor(x => x)
```

Comment:
better to put rules to separate method: void ConfigureRules() and call it from here